packages/arb-token-bridge-ui/src/components/TransferPanel/TokenRow.tsx

```diff
@@ -268,10 +289,10 @@ export function TokenRow({
       type="button"
       onClick={onClick}
       style={{ ...style, minHeight: '84px' }}
-      disabled={!tokenIsBridgeable}
+      disabled={!tokenIsBridgeable && !withdrawalModeArbOneUSDC}
```

Comment:
Thinking of keeping the JSX conditions as simple and compact as possible. 
The flag `tokenIsBridgeable` contains a combination of quite some conditions internally. We should handle `withdrawalModeArbOneUSDC` within the scope of `tokenIsBridgeable`.

ie. adding this 
``` 
const tokenIsBridgeable = useMemo(() => {
...
if (withdrawalModeArbOneUSDC) return true;
...
)}
```

Reply:
hm but the arb one USDC is not bridgeable so i don't think we should put it in the same var